src/components/picker/demos/demo2.tsx

```diff
@@ -1,39 +1,88 @@
 import React, { useState } from 'react'
-import { Button, Picker, Space } from 'antd-mobile'
+import { Button, Picker, Space, Toast } from 'antd-mobile'
+import type { PickerColumn } from 'antd-mobile/es/components/picker'
 import { DemoBlock, DemoDescription } from 'demos'
 import { basicColumns } from './columns-data'
+import { mockRequest } from './mockRequest'
 
 export default function () {
   const [visible, setVisible] = useState(false)
   return (
-    <DemoBlock title='自定义样式'>
-      <Space direction='vertical' block>
-        <Button
-          onClick={() => {
-            setVisible(true)
+    <>
+      <DemoBlock title='自定义样式'>
+        <Space direction='vertical' block>
+          <Button
+            onClick={() => {
+              setVisible(true)
+            }}
+          >
+            选择
+          </Button>
+          <DemoDescription>
+            你可以通过 CSS 变量对 Picker 的字号等进行自定义，CascadePicker 和
+            DatePicker 也同样支持自定义样式。
+          </DemoDescription>
+        </Space>
+        <Picker
+          style={{
+            '--title-font-size': '13px',
+            '--header-button-font-size': '13px',
+            '--item-font-size': '13px',
+            '--item-height': '30px',
+          }}
+          defaultValue={['Wed', 'pm']}
+          columns={basicColumns}
+          visible={visible}
+          onClose={() => {
+            setVisible(false)
           }}
-        >
-          选择
-        </Button>
+        />
+      </DemoBlock>
+      <DemoBlock title='懒加载数据'>
+        <LazyLoadColumnsDemo />
+      </DemoBlock>
+    </>
+  )
+}
+
+function LazyLoadColumnsDemo() {
+  const [visible, setVisible] = useState(false)
+  const [columns, setColumns] = useState<PickerColumn[]>([])
+  const [loading, setLoading] = useState(false)
+
+  const handleClick = async () => {
+    setVisible(true)
+    if (!columns.length && !loading) {
+      try {
+        setLoading(true)
+        const data = await mockRequest({ delay: 2000 })
+        setColumns(data)
+      } catch (error) {
+        Toast.show('请求失败')
+      } finally {
+        setLoading(false)
+      }
+    }
+  }
+
+  return (
+    <>
+      <Space direction='vertical' block>
+        <Button onClick={handleClick}>懒加载数据</Button>
         <DemoDescription>
-          你可以通过 CSS 变量对 Picker 的字号等进行自定义，CascadePicker 和
-          DatePicker 也同样支持自定义样式。
+          你可以在Picker显示时发起异步请求获取数据，提供了默认的骨架屏loading样式，CascadePicker
```

Comment:
```suggestion
          你可以在 Picker 显示时发起异步请求获取数据，提供了默认的骨架屏 loading 样式
```
字母前后可以加个空格，看起来比较舒服.. 也算是一种规范

Reply:
我感觉这个 demo 还是复杂了.. 其实就是一个 Loading 属性而已... 

title：加载状态

```jsx
<Picker visible={visible} loading />
```

描述都能省了...

Reply:
额，难道不是详细点么，模拟一次用户完整获取数据loading的过程...